tests/unit/services/wizard-manager-test.ts

```diff
@@ -130,7 +130,7 @@ module('Unit | Service | wizard-manager', function (hooks) {
 
       this.service.selectStep(step2Id, true);
       await settled();
-      assert.equal(this.service.focusedStepId, step1Id);
+      assert.equal(this.service.focusedStepId, step2Id);
```

Comment:
I'm not sure why these test was passing before, but the change I made here makes sense to me so 🤷